MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -283,8 +283,6 @@ public override void Log(string Message)
 
         public override void Present()
         {
-            base.Present();
```

Comment:
Because it is a call of empty method. I recommend either mark Present in GamePlatform abstract or place other content from Present of OpenTKGamePlatform to GamePlatform itself.